webapp/src/blocks.tsx

```diff
@@ -1108,6 +1110,14 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         return this.filterBlocks(subns, this.nsMap[ns]) || [];
     }
 
+    private openSpriteWizard() {
```

Comment:
avoid referring to "SpriteWizard" since you're building a general purpose wizard infrastructure.